app/controlplane/pkg/data/ent/schema/workflow.go

```diff
@@ -35,7 +35,7 @@ type Workflow struct {
 func (Workflow) Fields() []ent.Field {
 	return []ent.Field{
 		field.String("name").Immutable(),
-		field.String("project").Optional(),
+		field.String("project"),
```

Comment:
What are we doing with the current data? Will we migrate it?

Reply:
There is no need to migrate since all fields, in fact, have a project name, enforced somewhere else in the stack, not in the DB until now